x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/planner/ExpressionTranslator.java

```diff
@@ -33,4 +34,9 @@ public static Query wrapIfNested(Query query, Expression exp) {
         }
         return query;
     }
+
+    public static FieldAttribute checkIsFieldAttribute(Expression e) {
+        Check.isTrue(e instanceof FieldAttribute, "Line {}: Expected [{}] to be FieldAttribute but was not.", e.sourceLocation(), e);
```

Comment:
The style of the error message should be consistent with the other uses of the `Check.isTrue` method in similar situations, ie `Expected a FieldAttribute but received [{}]`.